cmd/ingress-perf.go

```diff
@@ -86,8 +89,47 @@ func run() *cobra.Command {
 	return cmd
 }
 
+func getGitInfo() (string, string) {
+
+	// Get the last released tag info
+	cmdTag := exec.Command("git", "describe", "--tags", "--abbrev=0")
+	var outTag bytes.Buffer
+	cmdTag.Stdout = &outTag
+	errTag := cmdTag.Run()
+	gitTag := "Unknown"
+	if errTag == nil {
+		gitTag = strings.TrimSpace(outTag.String())
+	}
+
+	// Get the last Commit Hash
+	cmdCommit := exec.Command("git", "rev-parse", "--short", "HEAD")
```

Comment:
This is a bit tricky, since will require the `ingress-perf` binary to be in a git repo.
I think is better the kube-burner's logic which passes variable values in building time

See the full example at:
- https://github.com/cloud-bulldozer/kube-burner/blob/fbb94861614b547a906e41cd913fe8187581de16/Makefile#L11-L45
- https://github.com/cloud-bulldozer/kube-burner/blob/master/pkg/version/version.go
- https://github.com/cloud-bulldozer/kube-burner/blob/fbb94861614b547a906e41cd913fe8187581de16/cmd/kube-burner/kube-burner.go#L55-L65

Reply:
Sure, thanks for the feedback, lemme me tinker with this a bit.

Reply:
@rsevilla87 I was thinking, we can rework this a bit, here's my 2 cents:

1) Add a reusable version to go-commons
2) Import this here and refactor the logic
3) Rework kube-burner's version imports too

Reply:
I don't think we should have this is go-commons, as it's just a few lines

Reply:
> I don't think we should have this is go-commons, as it's just a few golang lines and a combination of git commands + go build flags


Right, I was thinking, we could get benefitted across k8s-netperf/ingress-perf if we have had it in go-commons
But ack, will update as suggested.